### PR TITLE
Peak Value in Array issue solved

### PR DESCRIPTION
**Pull request by krishna-NIT**
#198 Peak value in an Array  added in JAVA
all Issue Resolved,
returns true if prime & false if not prime
@rudrakshi99 @nakul-19 @plazzy99 please let me know if any changes are required

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@nakul-19 @rudrakshi99 
Please review it, More than 29 days gone

**Comment by krishna-NIT** [comment]
@yasharth291 Please Review my Code

File: Code/Java/peakvalueinarray.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+then it will retuen that Element.
+*/
+
+package peakval;
+import java.util.Scanner;
+
+public class peakvalueinarray {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Enter Size of Arraay");
```
Comment (yasharth291) [inline Code/Java/peakvalueinarray.java:16]: Array Spelling

**Review by yasharth291 — CHANGES_REQUESTED** [review]

**Review by krishna-NIT — COMMENTED** [review]
Done @yasharth291, Correction Done

**Pull request merged**